### DISCUSSION TITLE
ERROR: The SMTP server has unexpectedly disconnected.

### DISCUSSION THREAD

**Discussion by arjwalker** [postalserver/postal#2386] [Help with using Postal]
Hi All,

Ive got this error when testing the SMTP server from external testing sites but the email does hit the queue and gets sent out successfully. 

![image](https://user-images.githubusercontent.com/91224074/226437966-d5c271ae-0a97-4e4a-8530-ab906d94b2c9.png)

Any idea what this could be and how I can stop it? 

Thanks in advance!

**Comment by willpower232** [comment]
It might be worth monitoring the server to make sure it isn't running out of memory or whatever that might cause it to stop receiving.

It is also possible this is related to a few little bugs which have been patched since the last release so hopefully there is a new release tagged soon.

**Reply by arjwalker** [reply]
OK thanks - any idea what the recommended cpu/memory would be for around 3000 emails per day?

**Reply by willpower232** [reply]
unfortunately it isn't really possible to say exactly how much you require so just monitoring your server during busy periods will be required.

You may be interested in #324 which has some mysql server optimisations which may be relevant.
